python/lsst/afw/display/ds9.py

```diff
@@ -40,19 +40,14 @@
     except Exception as e:
         # No usable version of display_ds9.
         # Let's define a version of getDisplay() which will throw an exception.
-        e.args = ["%s (is display_ds9 setup?)" % e]
 
-        # TODO: once we abandon Python 2 use the following simpler code:
-        # def getDisplay(*args, exception=e, **kwargs):
-        #     raise exception
-        class _RaiseException(object):
-            def __init__(self, exception):
-                self.exception = exception
```

Comment:
Yes, or better yet `RuntimeError("%s (is display_d9 setup?)" % e)` so the exception type is more predictable.

Another obscure issue for Python 2 is that `str(exception)` can raise an exception, but it may not be worth worrying about that.